datasync/datasync.go

```diff
@@ -0,0 +1,10 @@
+package datasync
+
+import (
+	datasyncnode "github.com/vacp2p/mvds/node"
+)
+
+type DataSync struct {
+	*datasyncnode.Node
+	*DataSyncNodeTransport
```

Comment:
Not sure where this belongs, but could we add a comment on what DataSyncNodeTransport is exactly? Or a link to relevant doc